src/datastation/scripts/ingest_flow_start_import.py

```diff
@@ -15,7 +15,8 @@ def main():
     args = parser.parse_args()
     service_baseurl = config['ingest_flow']['service_baseurl']
 
-    start_import(service_baseurl, args.deposit_path, args.continue_previous, is_migration=False)
+    start_import(service_baseurl, args.deposit_path, is_batch=not args.single_deposit,
+                 continue_previous=args.continue_previous, is_migration=False, is_dry_run=False)
```

Comment:
But why always is_dry_run=False?

Reply:
I just made it work without changing the command line arguments.

Reply:
OK, minimal change, I get your point. I'll merge this. If you have time please do a PR to add the dry-run option.